TakeArguments.java

```diff
@@ -0,0 +1,10 @@
+class TakeArguments{
```

Comment:
Please format your code! Which editor do you use? IntelliJ or Notepad or something different?

Reply:
Notepad is used as the editor.

Reply:
I think IntelliJ would be a much better option from now on. Please use this editor. If you need any help setting it up or if you have any other questions, please do not hesitate to ask.

Reply:
Hi Raimund, Me and Vishnu are working in a team. He modified some files so please review and merge the pull request.